Source/WebCore/Modules/WebGPU/GPUSwapChainDescriptor.h

```diff
@@ -39,17 +41,25 @@ struct GPUSwapChainDescriptor : public GPUObjectDescriptorBase {
     {
         return {
             { label },
-            WebCore::convertToBacking(size),
-            sampleCount,
             WebCore::convertToBacking(format),
             convertTextureUsageFlagsToBacking(usage),
+            viewFormats.map([] (auto& viewFormat) {
+                return WebCore::convertToBacking(viewFormat);
+            }),
+            WebCore::convertToBacking(colorSpace),
+            WebCore::convertToBacking(compositingAlphaMode),
+            width,
+            height,
         };
```

Comment:
Can we move implementation into `.cpp`/`.mm` as much as possible?

Reply:
None of these Descriptor.h files have accompanying .cpp files, mostly just for my own sanity when dealing with all this stuff. If we're going to start giving Descriptors accompanying .cpp files, we should do it consistently for all of them, which is a pretty substantial change I think, and would probably best be done in another patch (probably after some additional discussion among the team, too)